media-libs/libuninameslist/libuninameslist-20091231-r1.ebuild

```diff
@@ -21,5 +21,5 @@ src_configure() {
 
 src_install() {
 	default
-	find "${ED}" -name '*.la' -exec rm -f {} +
+	find "${ED}" -name '*.la' -exec rm -f {} + || die
```

Comment:
this can be
`find "${ED}"/usr -name '*.la' -delete`
or even 
`find "${ED%/}"/usr -name '*.la' -delete`